### ISSUE TITLE
The configuration to loop video

### ISSUE THREAD

**Issue by NikKovIos** [MillmanY/MMPlayerView#41]
Is there a configuration to loop video playback?

**Comment by Jaydeepvadalia** [comment]
```
 shareDetailCell.playerView.set(url: url) { (status) in
                        switch status {
                        case .end:
                            shareDetailCell.playerView.player?.seek(to: CMTimeMake(0, 1))
                            shareDetailCell.playerView.player?.play()
                            break
                        default:
                            break
                        }
         }
```

**Comment by MillmanY** [comment]
Fix on 5.0.4
mmplayerLayer.repeatWhenEnd = ture
